kfs-core/src/main/resources/arizona-ApplicationResources.properties

```diff
@@ -0,0 +1,4 @@
+#UAF-59 / UAF-1668 Missing Invoice Number on DV

```

Comment:
Each module has its own properties file now. The DV messages should be in kfs-core/src/main/resources/edu/arizona/kfs/fp/fp-resources.properties. Remove the PREQ message, for now.